packages/@aws-cdk/aws-applicationautoscaling/lib/cron.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Helper class to generate Cron expressions
+ */
+export class Cron {
+
+  /**
+   * Return a cron expression to run every day at a particular time
+   *
+   * The time is specified in UTC.
+   */
+  public static dailyUtc(hour: number, minute?: number) {
+    if (minute === undefined) {
```

Comment:
`minute = minute || 0`?

Reply:
Mmmmmyeah. Don't like to do that for types that can take on falsy values, though in this case it wouldn't matter.